### DISCUSSION TITLE
How good is wasmer for a plugin system that require parallel execution?

### DISCUSSION THREAD

**Discussion by glyh** [wasmerio/wasmer#2833] [Q&A]
As title. Say someone decides to implement Emacs plugin system again with WASM, and design it to be parallel. Should he choose wasmer or wasmtime? And why?

**Comment by SamuraiCrow** [comment]
The WASI standard hasn't defined a way to do multithreading yet.  As such, neither WasmTime nor Wasmer can implement that standard using multithreading.
